### DISCUSSION TITLE
Easy way to query geometry infos such as vertices/triangles count?

### DISCUSSION THREAD

**Discussion by minigorille** [google/model-viewer#2336] [General]
Hi,

Is there an "easy" way to expose/query geometry infos such as vertices count or triangles count? I know it'spossible with three.js so was hoping they would be some way to access that infos with ModelViewer.
Also saw that the model editor can run the Kronos test that return those infos but looking for a simple way to implement that within the ModelViewer.

Let me know, thanks !

**Comment by minigorille** [comment]
any ideas?

**Reply by elalish** [reply]
No, that's outside of model-viewer's scope. You can fork us if you want to dig down into three.js (or hack down using `$scene`, but you'll have to be careful of breaking changes).

**Reply by minigorille** [reply]
Ah thanks! That may be beyond my technical ability but I'll dig into it and see.
